policy/executor/internal/execution_manager.go

```diff
@@ -173,6 +173,10 @@ func (em *executionManager) executeCodeBundle(codeBundle *llx.CodeBundle, props
 			Str("qrid", codeID).
 			Dur("duration", time.Since(startTime)).
 			Msg("finished query execution")
+		if time.Since(startTime) > 5*time.Minute {
+			// if the query duration was more than 5 minutes, send an alert to platform
+			health.ReportSlowQuery("cnspec", cnspec.Version, cnspec.Build, health.SlowQueryInfo{Query: codeID, Duration: time.Since(startTime)})
```

Comment:
if we have the mql, it might make life easier to

Reply:
i couldnt find where to access the mql string from this context, but ya that would be awesome if we can